### DISCUSSION TITLE
c2v include paths, multiple mains, external libraries.

### DISCUSSION THREAD

**Discussion by sriram-srinivasan** [vlang/v#14918] [General]
Take Git, as an example. It is straightforward C, but relies on external libraries such as openssl. Further, the source directory has many CLIs that share a main.  How does one convey all this information to c2v? 

Even if I take a single file, say imap-send.c, I get the following:

```
C to V translator 0.3.1
  translating imap-send.c     ... In file included from imap-send.c:24:
In file included from ./cache.h:4:
./git-compat-util.h:353:10: fatal error: 'openssl/ssl.h' file not found
#include <openssl/ssl.h>
         ^~~~~~~~~~~~~~~
1 error generated.

```

How does one translate it? Is it possible to do it at this stage of the language evolution?

**Comment by medvednikov** [comment]
Include paths can be configured via the c2v.toml config

It's documented, also see how it's done in the doom example.

**Comment by sriram-srinivasan** [comment]
Thank you. My bad. I missed that.
